packages/@uppy/core/src/index.js

```diff
@@ -73,8 +73,7 @@ class Uppy {
           0: 'Select %{smart_count}',
           1: 'Select %{smart_count}',
         },
-        selectAllFilesFromFolderNamed: 'Select all files from folder %{name}',
-        unselectAllFilesFromFolderNamed: 'Unselect all files from folder %{name}',
+        allFilesFromFolderNamed: 'All files from folder %{name}',
```

Comment:
Again, trying to think if a change like this could be breaking: https://github.com/transloadit/uppy/pull/2397

Reply:
What if I merge this into `2.0` instead of `master`?

Reply:
Yeah, I think that would be safer.